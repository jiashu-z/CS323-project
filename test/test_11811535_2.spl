int test_11811535_2(){
  int a=0000, b=01, c=0x00, d=0x01, e=0, f=0x0;
}