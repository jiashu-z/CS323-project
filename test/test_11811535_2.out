Error type A at Line 2: unknown lexeme 0000
Error type A at Line 2: unknown lexeme 01
Error type A at Line 2: unknown lexeme 0x00
Error type A at Line 2: unknown lexeme 0x01